airbyte-webapp/src/components/ui/CodeEditor/CodeEditor.tsx

```diff
@@ -1,21 +1,57 @@
-import Editor from "@monaco-editor/react";
+import Editor, { Monaco } from "@monaco-editor/react";
 import React from "react";
 
+import styles from "./CodeEditor.module.scss";
+
 interface CodeEditorProps {
-  height?: string;
-  code: string;
+  value: string;
   language?: string;
+  theme?: "airbyte" | "vs-dark" | "light";
+  readOnly?: boolean;
+  onChange?: (value: string | undefined) => void;
+  height?: string;
+  lineNumberCharacterWidth?: number;
 }
 
-export const CodeEditor: React.FC<CodeEditorProps> = ({ code, height, language }) => {
+export const CodeEditor: React.FC<CodeEditorProps> = ({
```

Comment:
This component was modified to be more generic, since I am now using it in two places (YamlEditor and ConfigMenu)